react-design/my-app/src/App.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './App.css';
+import RotatingBanner from './RotatingBanner';
+
+const items = [
+  'Aardvark',
+  // 'Bengal',
+  // 'Caterpillar',
+  // 'Dromedary',
+  // 'Elephant',
+  // 'Ferret',
+];
```

Comment:
Don't comment out parts of your data set. Instead, modify your component to display only one item.